compose/ChartboostMediationDemoCompose/src/main/java/com/chartboost/mediation/sdk/demo/AdController.kt

```diff
@@ -161,15 +150,13 @@ object AdController : DefaultLifecycleObserver {
      * @param adType The type of ad to show.
      * @param placementName The placement name.
      * @param logState The log state to update.
-     * @param shouldUseFullscreenApi True if the fullscreen API should be used, false otherwise.
      * @param showBtnEnabled True if the corresponding show button should be enabled, false otherwise.
      */
     fun showAd(
         context: Context,
         adType: AdType,
         placementName: String,
         logState: MutableList<String>,
-        shouldUseFullscreenApi: MutableState<Boolean>,
         shouldUseFullscreenAdQueue: MutableState<Boolean>,
```

Comment:
I know this was not in scope but `shouldUseFullscreenAdQueue` is missing docs.

Reply:
https://github.com/ChartBoost/chartboost-mediation-android-sdk-demo/pull/39/commits/8ff6389acd4d2fd66016cd2365bc5d6131463bf1